engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -701,25 +707,32 @@ public void recompileShaders() {
     }
 
     @Override
-    public void initialise() { }
+    public void initialise() {
```

Comment:
@NicholasBatesNZ: which checkstyle are you using here, the one in IntelliJ or the one from the builds?

Reply:
Yes, I am running from source. I am using the Checkstyle-IDEA plugin configured to the terasology checkstlye.xml